iptables/iptables.go

```diff
@@ -61,19 +61,18 @@ type FirewallConfiguration struct {
 // https://github.com/istio/istio/blob/e83411e/pilot/docker/prepare_proxy.sh
 func ConfigureFirewall(firewallConfiguration FirewallConfiguration) error {
 
-	log.Printf("Tracing this script execution as [%s]\n", ExecutionTraceID)
+	log.Debugf("Tracing this script execution as [%s]", ExecutionTraceID)
 
-	startSection("current state")
+	log.Debugln("current state")
```

Comment:
Since we removed the separators, I'm not sure it's worth keeping this line in. It doesn't add much to the logs and it doesn't really do anything to set the context up.